### PR TITLE
Cr 648 core telephone capture development

### PR DESCRIPTION
**Pull request by PeterBochel**
Summary of changes:

- Now that rehearsal has completed I've been able to delete some code that was already available in the case api client.
- Replaced previous dummy launch endpoint with a real one, which returns a url whose payload has been build by the eqLauncher. 
- Application config extended to include keys used by eqLauncher

### CODE REVIEW

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java [hunk 1]
```diff
@@ -236,6 +242,63 @@ public ResponseDTO reportRefusal(UUID caseId, RefusalRequestDTO requestBodyDTO)
...
+    SingleUseQuestionnaireIdDTO newQuestionnaireIdDto =
+        caseServiceClient.getSingleUseQuestionnaireId(caseId, individual, individualCaseId);
+    log.with("newQuestionnaireID", newQuestionnaireIdDto.getQuestionnaireId())
+        .info("Have generated new questionnaireId");
+
+    // Finally, build the url needed to launch the survey
+    String encryptedPayload = "";
+    try {
+      String accountServiceUrl = null;
+      String accountServiceLogoutUrl = "https://" + appConfig.getDomain() + "/questionnaireSaved";
```
Comment (philwhiles) [inline src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java]: As discussed? both urls should be null

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/config/AppConfig.java [hunk 2]
```diff
@@ -15,5 +17,8 @@
   // private Rabbitmq rabbitmq;
   private AddressIndexSettings addressIndexSettings;
   private CaseServiceSettings caseServiceSettings;
+  @NotBlank private String domain;
```
Comment (philwhiles) [inline src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/config/AppConfig.java]: domain not needed

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -146,5 +143,25 @@ queueconfig:
   event-exchange: events
   dead-letter-exchange: events.deadletter.exchange
   
+domain: localhost
```
Comment (philwhiles) [inline src/main/resources/application.yml]: not needed

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java (hunk 1 above)
Comment (PeterBochel) [reply src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java]: Oooops. Both null now.

File: src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/CaseEndpointCaseLaunchTest.java [hunk 4]
```diff
@@ -69,19 +69,34 @@ public void getCaseById_GoodId() throws Exception {
...
-    ResultActions actions = mockMvc.perform(getJson("/cases/" + uuid + "/launch"));
+    ResultActions actions = mockMvc.perform(getJson("/cases/" + uuid + "/launch?individual=true"));
+    actions.andExpect(status().isBadRequest());
+  }
+
+  @Test
+  public void getCaseById_GoodIdBadAgent() throws Exception {
+    ResultActions actions =
+        mockMvc.perform(getJson("/cases/" + uuid + "/launch?agentId=ABC45&individual=true"));
     actions.andExpect(status().isBadRequest());
```
Comment (philwhiles) [inline src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/CaseEndpointCaseLaunchTest.java]: The test methods throughout this class need to be renamed to reflect that they are testing eq launch and not get case

File: src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/config/AppConfig.java (hunk 2 above)
Comment (PeterBochel) [reply src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/config/AppConfig.java]: Removed

File: src/main/resources/application.yml (hunk 3 above)
Comment (PeterBochel) [reply src/main/resources/application.yml]: Removed

File: src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/CaseEndpointCaseLaunchTest.java (hunk 4 above)
Comment (PeterBochel) [reply src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/CaseEndpointCaseLaunchTest.java]: Agree. Method names updated.

**Pull request merged**